### PR TITLE
Add nodeinstance connection message

### PR DESCRIPTION
**Pull request by lvrach**
The agent is responsible to send this message:
1. After agent connection for every node instance hosted in the agent that is at least queryable
2. Every time the node instance liveness or queryable status changes, either because streaming is disrupted or a database rotation removed all data of node instance.

Note: The agent should not send this message before a graceful or ungrateful agent disconnection.

### CODE REVIEW

**Review by nktsitas — APPROVED** [review]

File: proto/nodeinstance/v1/connection.proto [hunk 1]
```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+option go_package = "nodeinstance/v1;connection";
```
Comment (nktsitas) [inline proto/nodeinstance/v1/connection.proto:2]: should we follow `node`'s structure and have this like `nodeinstance/connection/v1;nodeinstanceconnection` ?

On that note, I've also opened https://github.com/netdata/aclk-schemas/pull/6 to fix the `_` inconsistency

File: proto/nodeinstance/v1/connection.proto [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+package nodeinstance.v1;
+
+import "google/protobuf/timestamp.proto";
+
+message UpdateNodeInstanceConnection {
```
Comment (papazach) [inline proto/nodeinstance/v1/connection.proto:8]: [question] Would it make sense to also include the `hops` in the payload?
Or hops for a nodeInstance should be computed every time in real time? 🤔 

https://dbdiagram.io/d/5efcac1d0425da461f041f33

Comment (papazach) [reply proto/nodeinstance/v1/connection.proto:8]: If there is no point for the `hops` to be included in the payload ☝🏼 
I think we're good with this definition (and the `NodeInstanceConnectionUpdated` one) and that we could proceed merging it @nktsitas @lvrach

**Review by chrikar — APPROVED** [review]

**Review by papazach — APPROVED** [review]
lgtm 👍🏼

**Review by underhood — APPROVED** [review]

**Pull request merged**